frontend/css/main.less

```diff
@@ -283,13 +310,28 @@ pre code.hljs {
     > :not(:first-child) {
       color: @orange;
     }
+
     margin-right: 0.7em;
   }
 }
 
 // For cover art in alerts
-@alertThumbnailSize: 60px;
-.alert .alert-thumbnail {
+@alertThumbnailSize: 40px;
```

Comment:
Was changing the size here required?

Reply:
I felt the thumbnail was a bit too big for the alert box and wanted to give more space to the text instead.